File: src/main/java/no/nav/arbeidsgiver/sykefravarsstatistikk/api/applikasjon/domenemodeller/Statistikkategori.java
package no.nav.arbeidsgiver.sykefravarsstatistikk.api.applikasjon.domenemodeller;

public enum Statistikkategori {
  LAND,
  SEKTOR,
  NÆRING,
  BRANSJE,
  VIRKSOMHET,
  OVERORDNET_ENHET,
  NÆRING5SIFFER, // deprecated -- bruk NÆRINGSKODE i stedet
  NÆRINGSKODE
}

File: src/main/java/no/nav/arbeidsgiver/sykefravarsstatistikk/api/applikasjon/domenemodeller/Statistikkategori.kt
package no.nav.arbeidsgiver.sykefravarsstatistikk.api.applikasjon.domenemodeller

enum class Statistikkategori {
    LAND,
    SEKTOR,
    NÆRING,
    BRANSJE,
    NÆRING5SIFFER,
    OVERORDNET_ENHET,
    VIRKSOMHET,

    // deprecated -- bruk NÆRINGSKODE i stedet
    NÆRINGSKODE
}
